lib/lambda-worker/lambda-worker.ts

```diff
@@ -107,13 +119,25 @@ export class LambdaWorker extends cdk.Construct {
       runtime: lambda.Runtime.NODEJS_14_X,
     });
 
+    if (props.lambdaProps.policyStatements) {
+      for (const statement of props.lambdaProps.policyStatements) {
+        lambdaWorker.role?.addToPolicy(statement);
+      }
+    }
+
     // Add main queue and DLQ as event sources to the lambda
     // By default, the main queue is enabled and the DLQ is disabled
     lambdaWorker.addEventSource(
-      new eventSource.SqsEventSource(lambdaQueue, { enabled: true })
+      new eventSource.SqsEventSource(lambdaQueue, {
+        enabled: true,
+        batchSize: 1,
```

Comment:
Always set the batch size to one. We need to be able to replay individual failures.